packages/main/src/TabContainer.js

```diff
@@ -226,6 +227,44 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the background color of the Tab Container's header.
+		 *
+		 * <br><br>
+		 * Available options are:
+		 * <ul>
+		 * <li><code>Solid</code></li>
+		 * <li><code>Transparent</code></li>
+		 * </ul>
+		 *
+		 * @type {TabContainerBackgroundDesign}
```

Comment:
There was a recent change and now we write the types with the `@alias` as defined in the JSDoc of the Type. Here is an example https://github.com/SAP/ui5-webcomponents/blob/main/packages/main/src/TabContainer.js#L143.
Try to rebase more often to always build on the most up-to-date codebase.